src/intro-to-c/arrays/arrays-intro.md

```diff
@@ -39,14 +45,21 @@ However, in our previous example, we only created the _skeleton_ that can hold o
 Currently, our array (essentially) looks like this:
 
 ```
-+------+------+------+------+------+
-|      |      |      |      |      |
-+------+------+------+------+------+
+   grades[0]    grades[1]    grades[2]    grades[3]    grades[4]
++------------+------------+------------+------------+------------+
+|            |            |            |            |            |
++------------+------------+------------+------------+------------+
 ```
 
-It's empty and barren. Let's put some grades into it!
+It's empty* and barren. Let's put some grades into it!
+
+_*Technically, when we instantiate an array like this, our array isn't really empty. Instead,
+it's filled with garbage values. We will learn more about this later, but it is important
+to know, since accessing these array values when we have garbage values still in them may give unintended behavior._
```

Comment:
The output of this may look a little funny with the asterisks here. You may want to escape the asterisks like "\*", or maybe try using a footnote as follows:

```suggestion
It's empty[^garbagedata] and barren. Let's put some grades into it!

[^garbagedata]: Technically, when we instantiate an array like this, our array isn't really empty. Instead, it's filled with garbage values. We will learn more about this later, but it is important to know, since accessing these array values when we have garbage values still in them may give unintended behavior.
```

There are a few other ways you may want to try, I'm just thinking about how to make this side note visually distinct/out of the way for the reader. (Hopefully striking the balance between too much information and gaps in information)